deploy/kubernetes/README.md

```diff
@@ -0,0 +1,21 @@
+# PD CSI Driver Deployment in Kubernetes
+
+## Kustomize Structure
+
+The current structure for kustomization is as follows
+
+* `base`: it contains the setup that is common to different driver versions.
+  * `controller_setup`: includes cluster setup and controller yaml files.
+  * `node_setup`:
+    * Linux: includes node yaml file and related setting that is only applicable for Linux.
+    * Windows: includes node yaml file and related setting that is only applicable for Windows.
+* `images`: it has a list of images for different versions.
+  * `stable`: image list of a stable driver release. Currently only has image list for Linux stable version.
+  * `alpha`: image list containing features in development, in addition to images in stable. It also includes Windows images.
```

Comment:
maybe mention the reason windows images are only included here because the support for windows is still alpha?

Reply:
Added a line up top